README.md

```diff
@@ -167,11 +167,11 @@ _Note_: A preset setting that does not match a predefined preset will be ignored
 ### Reusing specs from a collection of plugins
 
 **LazyVim** can be used without loading its options, autocommands and keymappings.
-The settings of the resulting configuration will default to stock neovim.
+The settings of the resulting configuration will default to stock Neovim.
 
 This can be useful during testing or when reporting an issue for one of the plugins.
 Instead of adding the full spec to a reproducible configuration,
-the spec as defined in **LazyVim** can be used.
+the spec as defined in **LazyVim** collection can be used.
```

Comment:
perhaps: **LazyVim**'s collection?

or: 
the spec provided by **LazyVim** can be used

Reply:
Maybe I'm misunderstanding your initial use of words, but it seemed to me like you meant the spec of Lazyvim itself like here https://github.com/abeldekat/lazyflex.nvim/blob/c43b3e5dde14b9cd4be974fc4c56a822640e9cbe/README.md?plain=1#L39 whereas I thought maybe you should have meant the Lazyvim collection as defined in `lazyflex` itself like here https://github.com/abeldekat/lazyflex.nvim/blob/c43b3e5dde14b9cd4be974fc4c56a822640e9cbe/README.md?plain=1#L189 Because you're mentioning just before
> Instead of adding the full spec to a reproducible configuration

So, my guess was that the purpose of your initial wording was, since we don't want to add the full spec to a reproducable configuration, the Lazyvim collection as defined in `lazyflex` should be used instead for more granual control. Did I misunderstand your initial wording?

Reply:
No, you're right.

> The spec as defined in LazyVim collection

I am thinking about a delimiter between the words LazyVim and collection. 
- LazyVim's collection
- the collection provided by LazyVim
- in the LazyVim collection

Instead of adding the full spec of a plugin to a reproducible configuration, the spec included in the LazyVim collection can be used.

Reply:
How about `in the LazyVim collection provided by lazyflex`?

Reply:
So it would be like
```
Instead of adding the full spec to a reproducible configuration, the spec as defined in the
 **LazyVim** collection provided by `lazyflex` can be used for more granual control.
```

Reply:
> Instead of adding the full spec of a plugin to a reproducible configuration, the spec included in the LazyVim collection can be used.

I'm sorry, I missed that when I read your previous comment. I only concentrated in the bullet points. Let's go with that.

Reply:
Great!

Reply:
>Instead of adding the full spec of a plugin to a reproducible configuration, the spec >included in the LazyVim collection can be used.

How about ... can be referenced

Reply:
or ... is referenced

Reply:
I believe `used` is a better word, since we are using the Lazyvim collection provided by `lazyflex` to make changes (filter out some stuff) to the default LazyVim spec. We are not making a reference to the LazyVim collection. Although, that's too nitpicking in my opinion :stuck_out_tongue: